lib/ts/config.ts

```diff
@@ -496,11 +522,23 @@ export async function getQuestions(flags: UserFlags) {
                 return "Invalid project name: " + validations.problems![0];
             },
         },
+        {
+            name: "ui",
+            type: "list",
+            message: "Choose the ui build type for your frontend:",
+            choices: mapOptionsToChoices(uiBuildOptions),
+            when: flags.ui === undefined,
```

Comment:
we no longer should need to use flag. Just answers, since we modify answers based on flag before calling prompt
?

Reply:
Fixed this, now it looks at the `naswers` instead of the user `flag`